javatests/arcs/core/analysis/EntityTypeOperationsTest.kt

```diff
@@ -89,9 +112,11 @@ class EntityTypeUnionTest {
         val numberSchema = Schema(setOf(SchemaName("Example")), numberField, "")
         val textEntity = EntityType(textSchema)
         val numberEntity = EntityType(numberSchema)
+
         val result = requireNotNull((textEntity union numberEntity).getOrNull())
+
         assertThat(result.entitySchema).isEqualTo(
-            (numberSchema union textSchema).getOrNull()
+            (numberSchema union textSchema).getOrNull()!!
```

Comment:
Do we need `!!`? Wouldn't `isEqualTo` take care of null values? Or, do you want to also make sure that `result.entitySchema` is not null? I know I wrote this code, but now that I look at it, it might be better to explicitly create the `numberSchema union textSchema` type and use it in the check:
```
val textNumberSchema = Schema( 
      setOf(SchemaName("Example"),
      mapOf("text" to FieldType.Text, "number" to FieldType.Number)
      emptyMap()
)
```

Reply:
Yep, the !! was to check that the value isn't null. 

Done.